src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum Figures {
```

Comment:
```suggestion
public enum FigureType {
```
множина для enum не вживається зазвичай